WordPress/Classes/ViewRelated/NUX/SignupGoogleViewController.swift

```diff
@@ -7,7 +7,7 @@ class SignupGoogleViewController: LoginViewController {
     // MARK: - Properties
 
     private var hasShownGoogle = false
-    @IBOutlet var titleLabel: UILabel?
+    @IBOutlet var titleLabel: UILabel!
```

Comment:
I bet you could decouple this as-is without changing the `UILabel?` to `UILabel!` 😜

Reply:
We're soooooo gonna have a long, very, so very long chat on the subject Nate..!!. But you're right, this isn't a decoulping-Y thing, so... for now, we'll do it your way.

**For now** 😃